content/millennium/r4/foundation/documents/document-reference.md

```diff
@@ -81,6 +84,20 @@ _Implementation Notes_
 
 <%= disclaimer %>
 
+### Example: search by encounter filters patients level documents
+
+#### Request
+
+    GET https://fhir-ehr.cerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/DocumentReference?patient=823932&encounter=863887
```

Comment:
Replace the dev tenant id with c1941 tenant "ec2458f2-1e24-41c8-b71b-0e701af7583d" at all places

Reply:
changes addressed